### PR TITLE
feature/parse-letters

### PR DESCRIPTION
**Pull request by builtbywill**

### CODE REVIEW

File: src/latex2sympy/latex2sympy.py [hunk 1]
```diff
@@ -878,14 +871,14 @@ def get_differential_var(self, d):
         return sympy.Symbol(text, real=True, positive=True)
 
     def get_differential_var_str(self, text):
-        for i in range(1, len(text)):  # pragma: no cover - loop break not recognized correctly
+        for i in range(15, len(text)):  # pragma: no cover - loop break not recognized correctly
```
Comment (lily514) [inline src/latex2sympy/latex2sympy.py:874]: @builtbywill 1 -> 15 on purpose?

Comment (lily514) [reply src/latex2sympy/latex2sympy.py:874]: or is that from the num of characters is \\differentialD?

Comment (builtbywill) [reply src/latex2sympy/latex2sympy.py:874]: @lily514 yeah that was the reason for the change, but looking at the coverage results, doesn't look like the method is called right now 🤔 i'm gonna take another glance at convert_frac and handle_integral and see if i can clean this up a smidge

Comment (builtbywill) [reply src/latex2sympy/latex2sympy.py]: @lily514 updated!

File: src/latex2sympy/latex2sympy.py [hunk 2]
```diff
@@ -866,20 +862,12 @@ def handle_ceil(self, expr):
...
-                idx = i
-                break
-        text = text[idx:]
-        if text[0] == '\\':
-            text = text[1:].strip()
-        return text
+    def is_differential_var(self, expr):
+        return isinstance(expr, sympy.Symbol) and len(expr.name) > 14 and expr.name[:14] == 'differentialD-'
+
+    def get_differential_var(self, expr):
```
Comment (lily514) [inline src/latex2sympy/latex2sympy.py:868]: @builtbywill I like this better

**Review by lily514 — APPROVED** [review]

**Comment by lily514** [comment]
@builtbywill conflicts?

**Comment by builtbywill** [comment]
> @builtbywill conflicts?

@lily514 conflicts resolved! thanks for the heads up

**Review by tragessere — APPROVED** [review]

**Pull request merged**